tools/install.py

```diff
@@ -226,6 +226,23 @@ def wanted_v8_headers(files_arg, dest):
     files_arg = [name for name in files_arg if name in v8_headers]
     action(files_arg, dest)
 
+  def wanted_zoslib_headers(files_arg, dest):
+    zoslib_headers = [
+      zoslibinc + '/csrsic.h',
+      zoslibinc + '/edcwccwi.h',
+      zoslibinc + '/zos-base.h',
+      zoslibinc + '/zos-bpx.h',
+      zoslibinc + '/zos-char-util.h',
+      zoslibinc + '/zos-io.h',
+      zoslibinc + '/zos-semaphore.h',
+      zoslibinc + '/zos-setlibpath.h',
+      zoslibinc + '/zos-sys-info.h',
+      zoslibinc + '/zos-tls.h',
+      zoslibinc + '/zos.h',
```

Comment:
Would it be an issue if we add more header files to zoslib later down the road? Would we just have to create a new PR?

Reply:
Can we use something like https://docs.python.org/3/library/glob.html to include all files with a `.h` extension?